astropy/table/jsviewer.py

```diff
@@ -26,7 +26,7 @@ class Conf(_config.ConfigNamespace):
         'The URL to the jquery datatables library.')
 
     css_urls = _config.ConfigItem(
-        ['https://code.jquery.com/ui/1.11.1/themes/overcast/jquery-ui.css'],
+        ['https://cdn.datatables.net/1.10.2/css/jquery.dataTables.css'],
```

Comment:
Version 1.10.9 of DataTables is available. Also it seems a good occasion to upgrade jquery to 1.11.3